.github/workflows/build_conda_pkg.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           DOCKERHUB_USER: ${{ secrets.DOCKERHUB_USER }}
           DOCKERHUB_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
         run: |
-          git clone -b latest_release_changes --single-branch https://github.com/conda-forge/evalml-core-feedstock
+          git clone -b ww-update-branch --single-branch https://github.com/conda-forge/evalml-core-feedstock
```

Comment:
I can change this back but the conda check will be red on this pr. Just fyi that might mean that @dsherry needs to merge this in.

Reply:
@freddyaboulton got it. What do you want to do?

If we're gonna merge it red, please write up an explanation of why its red so others can follow along.

Reply:
Sorry if I missed this, why can't we add the woodwork upgrade to `latest_release_changes`?